Classes/Tca/Field/RteField.php

```diff
@@ -15,15 +15,15 @@ protected function configureOptions(OptionsResolver $resolver)
 
         $resolver->setDefaults([
             'dbType' => "MEDIUMTEXT DEFAULT NULL",
-            'richtextConfiguration' => 'default',
+            'richtextConfiguration' => null,
         ]);
 
-        $resolver->setAllowedTypes('richtextConfiguration', 'string');
+        $resolver->setAllowedTypes('richtextConfiguration', ['string', 'null']);
```

Comment:
Is there a use-case where it is needed to set this to null? the if-condition would skip this anyway